src/e2e_test/adaptive_export_loadtest/harness/ae_vs_all.sh

```
@@ -0,0 +1,32 @@
+# ae_vs_all.sh — AE capture-fraction A/B: FILTER (anomaly-gated) vs EVERYTHING (passthrough),
+# SAME fixed load, back-to-back. Runs NODE-SIDE on the rig dev-machine (kubectl is local —
+# no labctl, no tailnet). forensic_db active-part deltas (rows+bytes) per table per arm.
+# capture fraction = AE/ALL. Requires live kubescape (fresh anomalies → FILTER writes flagged pods).
+set -uo pipefail
+NS=log4j-poc; CHPOD=chi-forensic-soc-db-soc-cluster-0-0-0
+WIN=${WIN:-150}; WARM=${WARM:-60}; OUT=/tmp/ae_vs_all.txt
+chq(){ kubectl -n clickhouse exec "$CHPOD" -c clickhouse -- clickhouse-client -q "$1" 2>/dev/null; }
+snap(){ chq "SELECT table, sum(rows), sum(data_compressed_bytes) FROM system.parts WHERE database='forensic_db' AND active AND table IN ('http_events','dns_events','pgsql_events','conn_stats') GROUP BY table FORMAT TSV"; }
+FE=$(kubectl -n $NS get svc frontend -o jsonpath='{.spec.clusterIP}' 2>/dev/null)
+echo "frontend=$FE win=${WIN}s warm=${WARM}s $(date -u +%H:%M:%S)" | tee "$OUT"
+for i in $(seq 1 5); do kubectl -n $NS delete pod cl-$i --ignore-not-found --wait=false >/dev/null 2>&1; done
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)